src/frontend/admin/src/components/presentational/table/useTableResourcePagination.spec.tsx

```diff
@@ -0,0 +1,36 @@
+import { renderHook, waitFor } from "@testing-library/react";
+import { useTableResourcePagination } from "@/components/presentational/table/useTableResourcePagination";
+import { Organization } from "@/services/api/models/Organization";
+import { useOrganizations } from "@/hooks/useOrganizations/useOrganizations";
+import { TestingWrapper } from "@/components/testing/TestingWrapper";
+
+describe("useFetchData Hook", () => {
+  it("initial and success state", async () => {
```

Comment:
IMO this test describe isn't really clear about what it is expecting + doesn't fit with the `it ...` syntax.